src/main/scala/GuessWho/GuessWho.scala

```diff
@@ -2,21 +2,42 @@ package GuessWho
-  val beff: Character = Character(name = "Beff", hasHair = true, hasFacialHair = false, hasGlasses = true, hasHat = true, gender = Gender.FEMALE, eyeColour = EyeColour.GREEN, hairColour = HairColour.BRUNETTE)
+  private val jon: Character = Character(name = "Jon", hasHair = false, hasFacialHair = true, hasGlasses = true, hasHat = true, gender = Gender.MALE, eyeColour = EyeColour.BROWN, hairColour = HairColour.NONE)
+  private val tony: Character = Character(name = "Tony", hasHair = false, hasFacialHair = false, hasGlasses = true, hasHat = false, gender = Gender.MALE, eyeColour = EyeColour.GREEN, hairColour = HairColour.NONE)
+  private val bill: Character = Character(name = "Bill", hasHair = true, hasFacialHair = true, hasGlasses = false, hasHat = true, gender = Gender.MALE, eyeColour = EyeColour.BROWN, hairColour = HairColour.BRUNETTE)
+  private val paul: Character = Character(name = "Paul", hasHair = true, hasFacialHair = false, hasGlasses = false, hasHat = true, gender = Gender.MALE, eyeColour = EyeColour.GREEN, hairColour = HairColour.BRUNETTE)
+  private val gary: Character = Character(name = "Gary", hasHair = true, hasFacialHair = true, hasGlasses = false, hasHat = true, gender = Gender.MALE, eyeColour = EyeColour.BROWN, hairColour = HairColour.BLONDE)
+  private val steve: Character = Character(name = "Steve", hasHair = true, hasFacialHair = false, hasGlasses = true, hasHat = false, gender = Gender.MALE, eyeColour = EyeColour.BLUE, hairColour = HairColour.RED)
+  private val jess: Character = Character(name = "Jess", hasHair = true, hasFacialHair = false, hasGlasses = true, hasHat = false, gender = Gender.FEMALE, eyeColour = EyeColour.BLUE, hairColour = HairColour.BLONDE)
+  private val vicky: Character = Character(name = "Vicky", hasHair = true, hasFacialHair = false, hasGlasses = false, hasHat = true, gender = Gender.FEMALE, eyeColour = EyeColour.BROWN, hairColour = HairColour.RED)
+  private val sarah: Character = Character(name = "Sarah", hasHair = false, hasFacialHair = false, hasGlasses = false, hasHat = false, gender = Gender.FEMALE, eyeColour = EyeColour.BLUE, hairColour = HairColour.NONE)
+  private val beff: Character = Character(name = "Beff", hasHair = true, hasFacialHair = false, hasGlasses = true, hasHat = true, gender = Gender.FEMALE, eyeColour = EyeColour.GREEN, hairColour = HairColour.BRUNETTE)
 
   private val characterList:Seq[Character] = Seq(jon, tony, bill, paul, gary, steve, jess, vicky, sarah, beff)
 
-  val game:GameBoard = new GameBoard(characters = characterList, defaultChosenCharacter = None)
+  private val game:GameBoard = new GameBoard(characters = characterList, defaultChosenCharacter = None)
+
   private var _playersRemaining:Seq[Character] = game.getRemainingCharacters
 
 
+  def displayCharacters(characters:Seq[Character]):Unit = {
+    _playersRemaining.foreach { character =>
```

Comment:
Its usually best practice to override the `toString` method of the case class e.g.

```scala
case class Character(
                      name: String,
                      hasHair: Boolean,
                      hasFacialHair: Boolean,
                      hasGlasses: Boolean,
                      hasHat: Boolean,
                      gender:Gender,
                      eyeColour: EyeColour,
                      hairColour: HairColour
                    ) {
  override def toString: String =
    s"""
       |Name: $name
       |Has Hair: $hasHair
       |Has Facial Hair: $hasFacialHair
       |Has Glasses: $hasGlasses
       |Has Hat: $hasHat
       |Gender: $gender
       |Eye Colour: $eyeColour
       |Hair Colour: $hairColour
    """.stripMargin
}
```

This method would then become

```scala
def displayCharacters():Unit = {
    _playersRemaining.foreach { character =>
      println(
        character.toString
      )
    }
    println(s"Number of remaining characters: ${_playersRemaining.length}")
  }
```

This makes it more readable and allows you to reuse the method you have across the codebase